compiler/plc_driver/src/cli.rs

```diff
@@ -316,7 +328,9 @@ pub fn get_config_format(name: &str) -> Option<ConfigFormat> {
 
 impl CompileParameters {
     pub fn parse<T: AsRef<OsStr> + AsRef<str>>(args: &[T]) -> Result<CompileParameters, ParameterError> {
-        CompileParameters::try_parse_from(args).and_then(|result| {
+        CompileParameters::try_parse_from(args).and_then(|mut result| {
+            result.got_layout_file = Some(String::from("tmp.json"));
```

Comment:
This is a hack to force the GOT layout file to exist and to happen at a certain location. This can be avoided by instead making `plc` accept it from the `JSON` file it gets as an argument, but I did not have time to research this yet.

Reply:
I'll address this in a followup